<launch>
    <node pkg="joy" type="joy_node" name="joy" output="screen"/>
    <node pkg="pcv_base" type="teleop_twist_joystick.py" name="teleop" output="screen"/>
</launch>
